sdk/eventhub/eventhubs-checkpointstore-blob/samples/apiSpecificContainerClient.js

```diff
@@ -3,28 +3,35 @@
   Licensed under the MIT Licence.
 
   This sample demonstrates how to extend the Storage Blob Container Client
-  to downlevel the API version used when communicating with the service.
+  to change the API version used when communicating with the service.
 
   This may be useful if the environment you are targetting supports an
   older version of Storage Blob than is officially supported by the
   @azure/storage-blob SDK.
+
+  For example, if you are running Event Hubs on Azure Stack Hub,
+  the Azure Stack Hub version 2002 supports up to version 2017-11-09
+  of the Azure Storage service.
+
+  For more information on the Azure Storage versions supported on Azure Stack,
+  please refer to https://docs.microsoft.com/en-us/azure-stack/user/azure-stack-acs-differences?view=azs-1910.
```

Comment:
@conniey suggested that we remove `en-us` locale from the url to redirect the user to their default locale.

Reply:
That's a fantastic suggestion, will update.

Reply:
@chradek hey, please remove the url parameter that points to a specific version of the documentation (?view=azs-1910). Just use docs.microsoft.com/en-us/azure-stack/user/azure-stack-acs-differences
Using that would default to the latest version of Azure Stack and that is what we want. Thanks.

Reply:
@jfggdl I removed the specific version. I also updated the sample READMEs to add a table including where to find the sample that changes the storage API version. There's 2 READMEs because we're separating TypeScript and JavaScript.